string first_name
string last_name
int32 age
int32 score
string phone_number
int32 id_number
int32 angle
int32 speed
